src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,7 @@
-export const UserInfo = () => <>Put the user here</>;
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
```

Comment:
Use destruction for `user`

Reply:
you mean  `{ user }
const userName = user.name;
const userMail = user.email;

<a className="UserInfo" href={`mailto:${userMail}>
{userName}`